tests/CMakeLists.txt

```diff
@@ -138,21 +140,22 @@ endif()
 # INSTALLATION
 #----------------------------------
 
-set(TESTS_INSTALL_PATH "/opt/test/bin" CACHE STRING "Specifies where the test binaries will be installed")
+include(GNUInstallDirs)
+set(TESTS_INSTALL_PATH "tests" CACHE STRING "Specifies where the test binaries will be installed")
```

Comment:
Is `tests` dir a kind of standard in packaging? When installing into `/`, it would be `/tests`, when installing into `/usr`, it would be `/usr/tests`.

Reply:
As an additional note, changing the default install path may break already existing packagers. But I don't want to create a new major release just for that. I think such a change could be acceptable with migration from `v1.3` to `v1.4`. What do you think from your packager POV?

Reply:
`/usr/tests/<project>` is standard (per [hier(7)](https://man.netbsd.org/hier.7) and [tests(7)](https://man.netbsd.org/tests.7)) on systems using Kyua like NetBSD and FreeBSD. Otherwise, tests are not usually installed. Some projects install in random places like `/usr/share/<project>/tests` (see `CMAKE_INSTALL_DATADIR`) or `/usr/share/examples/<project>`.

`BUILD_TESTS` is disabled by default, so packagers are unlikely to build, let alone, install tests. Looking at [Repology](https://repology.org/project/sdbus-cpp/versions) only Debian touches it but [skips](https://salsa.debian.org/zhsj/sdbus-cpp/-/blob/master/debian/not-installed) installing `opt/test/bin` in favor of using [dedicated system bus instance](https://salsa.debian.org/zhsj/sdbus-cpp/-/blob/bfd17ed50a31451cfd19e01fca544756e5c1b397/debian/rules#L19-29).

Prebuilt tests are not part of public API, so changing the install location seems fine for a minor version update. Even moving tests installation from `BUILD_TESTS` to `INSTALL_TESTS` should be fine.

Reply:
Completely forgot, some projects install tests under regular `/usr/bin` (aka `${CMAKE_INSTALL_BINDIR}`) but gate based on off-by-default option like `INSTALL_TESTS`.

Reply:
I've dropped `/opt/test/bin` change to avoid scope creep. The priority here is to get FreeBSD support rather than simplify packaging.

Reply:
OK. Could you please submit a separate PR with proposed packaging simplifications? Including splitting into `BUILD_TESTS` and `INSTALL_TESTS` (and whatever else you are proposing to improve or simplify packaging of the library). Your contribution would be appreciated 👍 🙂